packages/dune-release/dune-release.1.5.2/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "2.0"
+synopsis: "Release dune packages in opam"
+description: """
+`dune-release` is a tool to streamline the release of Dune packages in
+[opam](https://opam.ocaml.org). It supports projects built
+with [Dune](https://github.com/ocaml/dune) and hosted on
+[GitHub](https://github.com)."""
+maintainer: ["Nathan Rebours <nathan.p.rebours@gmail.com>"]
+authors: [
+  "Daniel Bünzli"
+  "Thomas Gazagnaire"
+  "Nathan Rebours"
+  "Guillaume Petiot"
+  "Sonja Heinze"
+]
+license: "ISC"
+homepage: "https://github.com/ocamllabs/dune-release"
+bug-reports: "https://github.com/ocamllabs/dune-release/issues"
+depends: [
+  "dune" {>= "2.7"}
+  "ocaml" {>= "4.06.0"}
+  "curly"
+  "fmt" {>= "0.8.7"}
+  "fpath" {>= "0.7.3"}
+  "bos"
+  "cmdliner"
+  "re" {>= "1.7.2"}
+  "astring"
+  "opam-format" {>= "2.1.0~beta"}
```

Comment:
```suggestion
  "opam-file-format" {>= "2.1.1"}
  "opam-format" {>= "2.1.0~beta"}
```

uses OpamParserTypes.FullPos

Reply:
Is this the right thing to do, or we should instead add
```
conflicts: [
  "opam-file-format" {< "2.1.1"}
]
```
Ping @kit-ty-kate

Reply:
I created [this PR](https://github.com/ocamllabs/dune-release/pull/412) upstream, and I think given we use `OpamParserTypes` directly, we should declare a dependency on `opam-file-format.2.1.2` (I think it was added in 2.1.2).

Reply:
2.1.1 seems fine, at least the lower bound check is not failing now.
The remaining failures are due to the CI environment for certain distributions, e.g.
```
$ git commit -m "" --allow-empty-message --quiet
# +  error: Terminal is dumb, but EDITOR unset
# +  Please supply the message using either -m or -F option.
# +  [1]
```

Reply:
Ah yes, I assume that's because `OpamParserTypes` is now coming from... somewhere else. `opam-format` probably.

Looks like the git version in CentOS is so old, that it doesn't accept empty commit messages. I've [made a PR upstream](https://github.com/ocamllabs/dune-release/pull/413) but given we don't test on CentOS 7 upstream there is the question on whether supporting this release is worth the effort.